src/include/migraphx/op/unary.hpp

```diff
@@ -30,6 +30,7 @@
 #include <migraphx/argument.hpp>
 #include <migraphx/stringutils.hpp>
 #include <migraphx/value.hpp>
+#include <migraphx/operation.hpp>
```

Comment:
Dont include operation.hpp. We should just include the header for `dyn_output` struct.